hooks/textbook-toc-utils.ts

```diff
@@ -3,13 +3,10 @@ type TocType = Array<[string, string[]]>
 export { extractToc, formatTocLines, TocType }
 
 function extractToc (indexContent: string): TocType {
-  // Chapter titles are of form `Chapter X. Chapter title<`.
-  const allChapters = (indexContent.match(/Chapter\s+\d+\.\s+[^<]+/g) || [])
-    .map(entry => entry.trim())
-  // Topic titles are of form `X.Y <a ...>Topic Title<`
-  const allTopics = (indexContent.match(/(\d+.\d+\s+)<a[^>]+([^<]+)/g) || [])
-    .map(entry => entry.replace(/<a[^>]+>/, '').trim())
-
+  // Chapter titles are of form `X. Chapter title<`.
+  const allChapters = (indexContent.match(/>\s+\d+\.\s+[^<]+/g) || []).map(clean)
+  // Topic titles are of form `X.Y Topic Title<`
+  const allTopics = (indexContent.match(/>\s+\d+.\d+\s+[^<]+/g) || []).map(clean)
```

Comment:
Maybe we could create two RegExp:
```
const chapterRegex = new RegExp('>\\s+\\d+\\.\\s+[^<]+', 'g')  
const topicRegex = new RegExp('>\\s+\\d+.\\d+\\s+[^<]+', 'g')
```
an create a function to reuse the code

```
function getFromContent(regexToSearch, indexContent: string){
    return (indexContent.match(regexToSearch) || []).map(clean)
}
```
This is not blocking, it's just a suggestion